rules-java-archives/addon/src/main/java/org/jboss/windup/rules/apps/java/archives/config/ArchiveIdentificationConfigLoadingRuleProvider.java

```diff
@@ -36,9 +39,15 @@
         return Initialization.class;
     }
 
+    // @Inject private StaticArchiveIdentifier identifier;
```

Comment:
I wanted to ask how to get a particular instance of the class, and use CDI instead of static getInstance().